### ISSUE TITLE
"Error 1"  possibly related to .newbies.fasta

### ISSUE THREAD

**Issue by cooketho** [macmanes-lab/Oyster_River_Protocol#18]
I'm getting the following error when running the command shown below.
`make: *** [/lab/solexa_reddien/tom/sandbox/orp_test/assemblies/diamond/test.newbies.fasta] Error 1`

I've attached the fastq files if you want to try and reproduce it. Maybe there's something wrong with the fastq? In any case, it would be helpful if ORP could output a more informative error message.
[fastq_files.zip](https://github.com/macmanes-lab/Oyster_River_Protocol/files/2800607/fastq_files.zip)

```
/lab/pr/tc/ORP/oyster.mk main \
STRAND=RF \
MEM=15 \
CPU=1 \
READ1=test..r1.fastq \
READ2=test.r2.fastq \
RUNOUT=test \
SPADES2_KMER=39 \
-d
```

**Comment by cooketho** [comment]
Yes, it is a reduced dataset for test purposes. Still, it should be able to handle that right?

**Comment by macmanes** [comment]
One can make a reduced dataset that is not representative of any real dataset, and then it's reasonably for the software to fail. I think this is what is happening here. No real transcriptome dataset would contain reads from only 1 transcript. 

That being said, I will work on an enhancement that will provide a more informative error message that would cover this specific issue
